packages/common-sdk/src/web3/network/fetcher/simple-fetcher-impl.ts

```diff
@@ -107,6 +107,16 @@ export class SimpleAccountFetcher<T, FetchOptions extends SimpleAccountFetchOpti
     return result;
   }
 
+  populateAccounts<U extends T>(
+    accounts: ReadonlyMap<string, U | null>,
+    parser: ParsableEntity<U>,
+    now: number
+  ): void {
+    Array.from(accounts.entries()).forEach(([key, value]) => {
+      this.cache.set(AddressUtil.toString(key), { parser, value, fetchedAt: now });
```

Comment:
super nit: ``ReadonlyMap<string, U | null>`` implies ``key`` is string without toString method.